sale_invoice_plan/models/sale_invoice_plan.py

```diff
@@ -153,12 +154,13 @@ def _get_amount_invoice(self, invoices):
             amount_invoiced = sum(lines.mapped("price_subtotal"))
         return amount_invoiced
 
+    @api.depends("invoice_move_ids.state")
```

Comment:
@kittiu What's the impact of this on existing database in terms of upgrade performance?

Reply:
Really not sure about this. But I doubt that this will be relevant to update the module? Since state is not changing. Do you foresee problem?